infrastructure/terraform/common/alb/main.tf

```diff
@@ -166,3 +166,20 @@ resource "aws_alb_listener" "otlp_collector_app_lblistner" {
   }
 }
 
+# Route53 Record for ALB
+data "aws_route53_zone" "otlp_abl_dns" {
+  name = "${var.alb_dns_zone}."
+}
+
+resource "aws_route53_record" "otlp_alb_dns" {
+  name    = "otlp-collector.${data.aws_route53_zone.otlp_abl_dns.name}"
```

Comment:
These should be consistent and match up to what we have in docker-compose, I'll create a story to cleanup the compose file.